index.html

```diff
@@ -259,11 +259,12 @@
           </p>
           <ul>
             <li>
-              <code>http://fedora.info/definitions/repository#EmbedResources</code>:
-              Requires a server to include representations of any contained resources in the response.
+              <code>http://www.w3.org/ns/oa#PreferContainedDescriptions</code>:
```

Comment:
Let me put this another way: say you had a Fedora/LDP implementation that also supported the WA spec. How would `oa:PreferContainedDescriptions` be different than `fedora:PreferEmbeddedResources` (or whatever the IRI is called)? In my reading of the WA spec, the two Prefer IRIs would be (functionally) equivalent, which makes me very suspicious of the need to create a separate IRI managed by Fedora.